src/Stratis.Bitcoin.Features.Consensus/CoinViews/Coindb/LeveldbCoindb.cs

```diff
@@ -186,6 +221,20 @@ public void SaveChanges(IList<UnspentOutput> unspentOutputs, HashHeightPair oldB
             this.performanceCounter.AddInsertedEntities(insertedEntities);
         }
 
+        /// <inheritdoc />
+        public int GetMinRewindHeight()
+        {
+            // Trim all rewind blocks prior to last checkpoint.
+            using (var iterator = this.leveldb.CreateIterator())
+            {
+                iterator.Seek(new byte[] { rewindTable });
```

Comment:
`Seek` moves to the first key in the table yes?

Reply:
To the first key starting with `rewindTable`.